doc/source/user_guide/cli.rst

```diff
@@ -143,15 +142,15 @@ Or an instance with a given process id (PID):
 
         .. code:: pwsh-session
 
-            (.venv) PS C:\Users\user\pymapdl> launch_mapdl stop --pid 40952
+            (.venv) PS C:\Users\user\pymapdl> pymapdl stop --pid 40952
             Success: The process with PID 40952 and its children have been stopped.
 
     .. tab-item:: Linux
         :sync: key1
                 
         .. code:: console
 
-            (.venv) user@machine:~$ launch_mapdl stop --pid 40952
+            (.venv) user@machine:~$ pymapdl stop --pid 40952
             Success: The process with PID 40952 and its children have been stopped.
```

Comment:
```suggestion
            Success: The process with PID 40952 and its children has been stopped.
```
Because "process" is the subject (singular), "has" should be the verb.

Reply:
But we are stopping the process AND its childrens... So .. plural?

Reply:
> But we are stopping the process AND its childrens... So .. plural?

Those are direct objects and not the subject of the sentence.  I asked Co-Pilot and it says it depends on the intended meaning:

Grammar question: Which of these is correct?
The process with PID 40952 and its children has been stopped.
The process with PID 40952 and its children have been stopped.
Copilot
Your personal and company data are protected
Both sentences are grammatically correct, but they convey slightly different meanings:

“The process with PID 40952 and its children has been stopped.”
In this sentence, the verb “has been stopped” refers to the entire process (PID 40952) and its associated child processes. It implies that the entire group of processes has been terminated or halted.
“The process with PID 40952 and its children have been stopped.”
Here, the verb “have been stopped” applies separately to each individual child process along with the main process (PID 40952). It suggests that each process has been stopped independently.

Choose the one that best fits the context of your message. If you want to emphasize the collective termination of the entire group, use the first sentence. If you want to highlight individual terminations, opt for the second sentence. 📝

Reply:
Because I explicitly kill all the processes (parent and child) I think ``have`` is appropriate here.

Thank you a lot @PipKat !!